main.go

```diff
@@ -21,11 +20,6 @@ import (
 	"github.com/spf13/viper"
 )
 
-//nolint:gochecknoinits
```

Comment:
Does the `config` package still get correctly configured without running `.ImportMe()` in the initialiser?

Reply:
yes, because ImportMe is an empty method

Reply:
> because ImportMe is an empty method

Yes, but it's a syntax trick to make sure Go imports the `config` package, to ensure the implicit `init()` function is executed during initialisation:
https://github.com/StuffNZ/get-secrets/blob/master/config/config.go#L17

`init()` is written to automatically read-in config and set-up logging, but it won't do this if that package is not included in the build.
Unfortunately, this is a bit too "magic" and I really wish I hadn't chosen to writte it this way...

TBH, the more-common convention these days might be something like:
`main`:
```
var _ = config.ForceImport
```

`config`:
```
var ForceImport struct {}
```

Reply:
oh, I got want you mean.
yup, I think var _ = config.ForceImport in main method would be good

Reply:
but even var _ = config.ForceImport will force us to know the magic init and main method convention.
I created another PR just make it simple and invoke the config method directly

Reply:
> I created another PR just make it simple and invoke the config method directly

This probably makes sense for this particular tool.
I implemented this a little differently in https://github.com/mexisme/go-config so that you can theroetically initialise in `init()` or `main()`.

For context: as a generic config package, it can be a little tricky, as you want to make sure you set-up certain parts of the config (e.g. Logging) as early as possible.
Quite a few Go modules implement behaviour in `init()` functions, for example, and we'd still very-much like to have Logging configured before those, where possible.